test/test_ltp.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import unittest
+from mock import PropertyMock, Mock
+from ltp import LtpLogs
+
+
+TEST_LOG = """
+test_1: some log 1
+test_1: some log 2
+test_1: some log 3
+test_3: some log 1
+test_3: some log 2
+test_3: some log 3
+"""
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.DEBUG)
+formatter = logging.Formatter("%(asctime)s - %(name)s - %(levelname)s - %(message)s")
+ch.setFormatter(formatter)
+logger.addHandler(ch)
```

Comment:
I would remove all of this, and leave only this, to silence the logging during tests:

```
logger = logging.getLogger()
logger.setLevel(logging.ERROR)
```